### DISCUSSION TITLE
aboutme

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#39] [Announcements]
# aboutme

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

http://localhost:4000/aboutme.html

**Comment by Denghongjian233** [comment]
怎么进美团 该死 美团不要我

**Reply by ATQQ** [reply]
同学校招社招呀🧐

校招搏一搏还是很有机会 ☺️当然也看投的部门和 工作地点

**Reply by Denghongjian233** [reply]
真该死 （社招 22届破二本 技术算法经验都还在提升中

**Reply by ATQQ** [reply]
可以满2年经验再看看机会😃

**Reply by Denghongjian233** [reply]
已经两年了 实习一年

**Reply by xlcodes** [reply]
很难吗？买个小电驴，当天就可以上班！不说了，我快超时了～～～

**Reply by ATQQ** [reply]
哈哈，那我顺手推一个合作链接：https://123.meituan.com/v3

**Reply by Denghongjian233** [reply]
6

**Reply by zhouxun0803** [reply]
6

**Comment by Denghongjian233** [comment]
社招阿




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月27日(星期四) 中午11:42
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ATQQ/sugar-blog] aboutme (Discussion #39)





 
同学校招社招呀🧐
 
校招搏一搏还是很有机会 ☺️当然也看投的部门和 工作地点

**Comment by Justin3go** [comment]
发个言，感谢这段代码，解决了我博客中giscus在浏览器刷新后不显示的BUG

```js
const showComment = ref(true)
watch(
  () => route.path,
  () => {
    showComment.value = false
    setTimeout(() => {
      showComment.value = true
    }, 200)
  },
  {
    immediate: true
  }
)
```

**Reply by ATQQ** [reply]
我之前也是好像也是遇到的类似问题🤭

**Reply by Justin3go** [reply]
朋友你好，giscus跟随vitepress夜/白切换，也许可以试试这个属性：
```ts
import { useData } from "vitepress";
const { isDark } = useData();
```

这个暴露出的`isDark `是响应式的，意味这用户在点击切换按钮时就可以做到重新加载评论组件，而非重新刷新页面。

之前参考了你的代码，也许这[部分代码](https://github.com/Justin3go/justin3go.github.io/blob/master/docs/.vitepress/theme/components/comment.vue#L29)能帮助到你

**Reply by ATQQ** [reply]
这个我记一下下来看看，这逻辑应该是加了的，但是目前问题应该是没有响应式的生效，我下来再看一下 问题已知

**Reply by ATQQ** [reply]
@Justin3go 这个已经支持了，但是还不够完美 切换的时候需要重新加载评论

我在 [《Hello 算法》](https://www.hello-algo.com/)的网站也看见用的这个，但是能丝滑的切换，回头学习一下

**Reply by Justin3go** [reply]
可以👍，我也瞧瞧

**Reply by ATQQ** [reply]
实现换成 [@giscus/vue](https://github.com/giscus/giscus-component) 组件就可以嘞

下面是commit @Justin3go 

https://github.com/ATQQ/sugar-blog/commit/7363df96466815d887458f08687728f3af17b341#diff-711e52abf1aa6b6e8e2cb11e18aff2dabf4c5e3a52b067fb1407dfa5f7e43f30

**Reply by Justin3go** [reply]
nice，非常感谢！

**Comment by NIU-BIN** [comment]
请教一下，文章列表怎么获得和实现，我没找到相关api，还有需要知道文章的更新时间，目前是想着通过读取md文件的修改时间，但是不知道怎么获取，读写文件都是node相关的api，目前想自己做一个时间线的文章归档组件，卡住了，请指教

**Reply by ATQQ** [reply]
已私信沟通解决😁

**Reply by kimshift** [reply]
获取文章列表在主题源码的utils/node/theme文件里面的这个getArticles方法

**Comment by sweet77-zsx** [comment]
到此一游
